packages/lib-utils/src/utils/common-fetch.ts

```diff
@@ -65,10 +67,14 @@ export const commonFetchJSON = async <TResult>(
     timeout,
     isK8sAPIRequest,
   );
+  const responseText: string = await response.text();
```

Comment:
This will be implicitly typed as `string` since `response.text()` returns `Promise<string>`.

```suggestion
  const responseText = await response.text();
```